models/server-models.js

```diff
@@ -7,4 +9,11 @@ exports.selectTopics = (req, res, next) => {
     return db.query(queryString).then((result) => {
         return result.rows;
       });
+}
+
+exports.selectAllEndpoints= (req, res, next) => {
```

Comment:
You don't need to import in a json file, to convert it to a string, and then parse it back to json in your controller. You can just serve the json file back from the controller to the client